### DISCUSSION TITLE
Channel Rebalancing "Fee Ratio" column

### DISCUSSION THREAD

**Discussion by Redinpais** [cryptosharks131/lndg#155] [Q&A]
Greetings.

On Channel Rebalancing, what is the point in the "Fee Ratio" column besides the need to keep it smaller than MaxCost% ?

Is there any ideal value that should be displayed in this column?

Can we conclude that Higher values mean lower profitability?

Thank you!

**Comment by cryptosharks131** [comment]
Correct and a fee ratio of below 1 is required to ever refill a particular channel, since you must at minimum pay your peers fee on the last hop.
